pkg/master/master_test.go

```diff
@@ -470,7 +470,7 @@ func TestDiscoveryAtAPIS(t *testing.T) {
 		},
 	}
 
-	assert.Equal(3, len(groupList.Groups))
+	assert.Equal(4, len(groupList.Groups))
```

Comment:
Pretty sure this was bugged before.  Look a few lines up and you'll see where `expectGroupNames` is set to have four names.  `expectVersions` has four values.